src/main/java/com/asa/CRP/model/CustomerRepresentative.java

```diff
@@ -10,62 +13,149 @@
 import javax.persistence.Table;
 
 
+/**
+ * The Class CustomerRepresentative.
+ * 
+ * @author Shivani Panwar
+ * @version 1.0
+ */
 @Entity
 @Table(name = "c_representatives")
 public class CustomerRepresentative implements Serializable{
 	
-	/**
-	 * 
-	 */
+	/** The Constant serialVersionUID. */
 	private static final long serialVersionUID = 5372010752544109811L;
 
+	/** The id. */
 	@Id
 	@GeneratedValue(strategy = GenerationType.IDENTITY) 
-	@Column(name="id")
-	private int id;
+	@Column(name="id", unique = true)
```

Comment:
Change the id to what it was. It is a generated value and it cannot be a string. Hibernate manages it itself.